tests/logger.rs

```diff
@@ -13,7 +13,7 @@ mod test_module {
     #[test]
     fn test_logger() {
         // Line number of tracing::info! location
-        const INFO_LINE: u32 = 50;
+        const INFO_LINE: u32 = 51;
```

Comment:
What's the meaning of this constant?

Reply:
This is the line number in this file of the `tracing::info!` macro call in this test.